ofrak_core/ofrak/core/gzip.py

```diff
@@ -87,19 +92,16 @@ class GzipPacker(Packer[None]):
     """
 
     targets = (GzipData,)
-    external_dependencies = (PIGZ,)
 
     async def pack(self, resource: Resource, config=None):
         gzip_view = await resource.view_as(GzipData)
-
-        result = BytesIO()
-        with GzipFile(fileobj=result, mode="w") as gzip_file:
-            gzip_child_r = await gzip_view.get_file()
-            gzip_data = await gzip_child_r.get_data()
-            gzip_file.write(gzip_data)
-
+        gzip_child_r = await gzip_view.get_file()
```

Comment:
Why is this change necessary? What was wrong with the Packer as it was implemented?

Reply:
using `zlib` is much faster, avoids a bunch of unnecessary processing in GzipFile

test on Silesia corpus:
https://gist.github.com/alchzh/a46140e683ff5e05be8d6b027a4f3fba

almost 4x faster on "mozilla" which is a 50MB tarfile of a Linux application so probably most applicable for us (4.4s for GzipFile vs 1.2s for zlib)

Reply:
on that note, we should probably be using pigz for packing large files as well